RELEASE_CHECKLIST.md

```diff
@@ -29,6 +29,12 @@ If you are releasing a beta or a release candidate, no official changelog is nee
 1. Run `cargo test --workspace`.
 1. Make sure you have `npm` installed, and run `cargo build`.
 
+
+### Update the README with the latest help strings
+
+1. Run `cargo run -- help` and copy the output into the README.
```

Comment:
```suggestion
1. Run `cargo run -- help` and copy the output into the root README and `/installers/npm/README`.
```

Reply:
oh - good catch. i just moved this above the part where we say to run `cargo build` since that will automatically copy the root README to the npm README

Reply:
Ahh, I wasn't aware that's what was happening 👌